### DISCUSSION TITLE
How to use vuejs plugin in vuepress 2?

### DISCUSSION THREAD

**Discussion by phpiscute** [vuepress/core#472] [Q&A] [answered]
use plugin in vuejs, e.g. `Vue.use(MyPlugin)`, how to use `vuejs plugin` in vuepress 2?

I want to use [vuejs-loading-plugin](https://www.npmjs.com/package/vuejs-loading-plugin), because need loading feature in vuepress custum component.

ps: I already  see [vuepress plugin doc](https://v2.vuepress.vuejs.org/guide/plugin.html#community-plugin), but it can't resolve my problem.

**Comment by meteorlxy** [comment, marked as answer]
https://v2.vuepress.vuejs.org/advanced/cookbook/usage-of-client-app-enhance.html
